tests/command.functions.test.js

```diff
@@ -0,0 +1,29 @@
+const test = require('ava')
```

Comment:
This test was inside `dev.test.js`.
Since it tests a different command I moved it to a separate file.